pprof.go

```diff
@@ -42,12 +42,17 @@ type readlineUI struct {
 	rl *readline.Instance
 }
 
+func isTerminal() bool {
```

Comment:
It doesn't look like changes in this file are required?

Reply:
Reverted.